obstacle_stop_planner_refine/CMakeLists.txt

```diff
@@ -39,6 +42,23 @@ target_link_libraries(obstacle_stop_planner_node
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
+
+  find_package(ament_cmake_gtest REQUIRED)
+  include(test/cmake/ament_cmake_auto_gtest-extras.cmake)
+  file(GLOB_RECURSE test_files test/**/*.cpp)
+  ament_auto_add_gtest(test_obstacle_stop_planner
+    test/test_obstacle_stop_planner.test
+    ${test_files}
+  )
+  target_include_directories(test_obstacle_stop_planner
+    PRIVATE
+    ${OpenCV_INCLUDE_DIRS}
+    ${PCL_INCLUDE_DIRS}
+  )
+  target_link_libraries(test_obstacle_stop_planner
+    ${OpenCV_LIBRARIES}
+    ${PCL_LIBRARIES}
```

Comment:
Do you still need OpenCV and PCL dependencies?(Probably PCL is exported from `pcl_ros` or something?)

Reply:
I can remove OpenCV dependency but I still need PCL dependency. (ex. https://github.com/tier4/obstacle_stop_planner_refine/blob/test-refactoring/obstacle_stop_planner_refine/include/obstacle_stop_planner/obstacle_point_cloud.hpp#L19)

Reply:
@KeisukeShima 
hmm, PCL is exported here.
https://github.com/ros-perception/perception_pcl/blob/135a2c29e3f46904f586e963cc4a4ec3fe594a9c/pcl_ros/CMakeLists.txt#L196

If you remove PCL-related things from CMakelists.txt and package.xml, the build fails?

Reply:
@kenji-miyake I added `pcl_ros` instead of PCL. Then I can remove PCL-related things from CMakeLists.txt

Reply:
It's much simpler! :+1: